website/docs/sql-function-glossary/sql-aggregate-functions/sql-round.md

```diff
@@ -0,0 +1,60 @@
+---
+id: sql-round 
+title: SQL ROUND 
+description: The ROUND aggregate function allows you to round numeric fields or values in SQL to the number of decimal places of your choosing.
+slug: /sql-function-glossary/sql-round
+---
+
+<head>
+    <title>Working with SQL ROUND</title>
+</head>
+
+If you’re reading this, that probably means you’re a data person. And as a data person who’s likely modeling data for analytics use cases or analysis, you’re going to need to round data from time to time.
```

Comment:
Maybe explain what rounding is here?  Something about reducing the number of decimal places in a numeric field